kratos/processes/levelset_convection_process.h

```diff
@@ -371,6 +373,8 @@ class LevelSetConvectionProcess
 
     bool mElementTauNodal; 
 
+    bool mHIsCalculated = true;
```

Comment:
```suggestion
    bool mCalculateNodalH = true;
```
`mHIsCalculated` is way misleading IMO (what comes to my mind is exactly the contrary to current behavior).